packages/custom_lint/example/analysis_options.yaml

```diff
@@ -9,3 +9,7 @@ linter:
     public_member_api_docs: false
     avoid_print: false
     unreachable_from_main: false
+
+custom_lint:
```

Comment:
I don't think that is useful to have in the default example

Reply:
I will remove it :)